server/config.ru

```diff
@@ -0,0 +1,35 @@
+require "roda"
+
+class App < Roda
+  route do |r|
+    r.root do
+      r.redirect "/outlier/1"
+    end
+
+    r.get "slow_service" do
+      sleep(1)
```

Comment:
Maybe?
```suggestion
      sleep(p(arams[:sleep_time] || 1).to_i)
```

Reply:
It should be something like

```
sleep(r.params.fetch("sleep_time", 1).to_i)
```

`r.params["sleep_time"]` returns nil or a string. We cannot invoke sleep with a string argument, so we need to convert it to an integer.